test/src/go.mod

```diff
@@ -1,9 +1,58 @@
 module github.com/cloudposse/terraform-aws-lambda-function
 
-go 1.16
+go 1.18
 
 require (
 	github.com/aws/aws-sdk-go v1.40.56
 	github.com/gruntwork-io/terratest v0.40.0
```

Comment:
Update `terratest` to `v0.40.16` or later because of known issues with `v0.40.15`or earlier.